#162948949 Update TravisCI to run tests

JoshuaKodhe opened the pull request:
#### What does this PR do?

 - Integrate TravisCI and Github allowing TravisCI to run tests

#### Description of Task to be completed?

 - Enabled the GitHub project on TravisCI 
 - Creating and setting up the .travis.yml file with the necessary commands

#### How should this be manually tested?

Visit the https://github.com/andela/ah-fulldeck-frontend
 - Change the branch from the develop  branch to ch-update-travisCI-162948983
 - A badge should be present

#### What are the relevant pivotal tracker stories?

[#162948949](https://www.pivotaltracker.com/story/show/162948949)

#### Screenshots
<img width="980" alt="screenshot 2019-02-07 at 07 39 41" src="https://user-images.githubusercontent.com/43740193/52390843-9b03c300-2aab-11e9-9778-ec8a05c083ab.png">

Nicanor008 reviewed (commented):
Squash the two commit messages into one

JoshuaKodhe committed:
chore (travisCI): update TravisCI to run tests

- ensure that TravisCI runs the tests for our app
- add a build badge to the readme

[Delivers #162948949]

victorjambo commented on README.md:
```diff
@@ -0,0 +1,7 @@
+[![Build Status](https://travis-ci.org/andela/ah-fulldeck-frontend.svg?branch=ch-update-travisCI-162948949)](https://travis-ci.org/andela/ah-fulldeck-frontend)
```
@JoshuaKodhe Change the branch to develop

JoshuaKodhe replied:
> @JoshuaKodhe Change the branch to develop

@victorjambo I have changed the badge but the build is still unknown, changes for the badge will occur once the branch is merged into develop branch.

victorjambo commented:
@JoshuaKodhe change commit msg from `Starts` to `Delivers`

Nicanor008 commented on src/App.test.js:
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from './App';
```
Intelligent way of obtaining the test file from another branch by rebasing.

Nicanor008 reviewed (dismissed):
Intelligent way of obtaining the test file from another branch by rebasing.

victorjambo reviewed (approved):
LGTM
